docs/iana-application.md

```diff
@@ -0,0 +1,51 @@
+### Transport protocols
+- [x] TCP
+- [x] UDP
+- [ ] SCTP
+- [ ] DCCP
+
+### Service Name
+StatsD
+
+### Desired Port Number
+8125
+
+### Description
+Network daemon for collecting metrics
+
+### Reference
+StatsD is used to collect statistics such as counters and timers to measure system and application performance. StatsD receives metrics over UDP or TCP in a basic line protocl of the following format:
+
+`<metricname>:<value>|<type>`
+
+The metric name (or "bucket") is an identifier for the metric being sent. Values represent the metric value being pushed to statsd and should be of the specified "type". Statsd supports numerous mertric types such as counters, gagues, and timers. Each has it's own single character key which is used in the type section.
```

Comment:
Should this mention expansions that include Dogstatsd formats?

Reply:
Good idea, I forgot about dogstatsd, that'll help outline the stability of the protocol as well.